packages/plugin-web-attribution-browser/src/helpers.ts

```diff
@@ -17,13 +17,25 @@ const domainWithoutSubdomain = (domain: string) => {
   return parts.slice(parts.length - 2, parts.length).join('.');
 };
 
-export const isNewCampaign = (current: Campaign, previous: Campaign | undefined, options: Options) => {
+export const isNewCampaign = (
+  current: Campaign,
+  previous: Campaign | undefined,
+  options: Options,
+  isNewSession = false,
+) => {
   const { referrer, referring_domain, ...currentCampaign } = current;
   const { referrer: _previous_referrer, referring_domain: prevReferringDomain, ...previousCampaign } = previous || {};
 
   if (isExcludedReferrer(options.excludeReferrers, current.referring_domain)) {
     return false;
   }
+
```

Comment:
[Here](https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/plugin-web-attribution-browser/src/web-attribution.ts), we will add the domain into the exclude referrers, so direct traffic will return to false, and won't be treat as new campaign.